terraform/modules/services/forecast_pvsite/ecs.tf

```diff
@@ -0,0 +1,49 @@
+# define aws ecs task definition
+# needs access to the internet
+
+resource "aws_ecs_task_definition" "ecs-task-definition" {
+  family                   = var.app-name
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+
+  # specific values are needed -
+  # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+  cpu    = 1024
+  memory = 7168
+
+  task_role_arn      = aws_iam_role.app-role.arn
+  execution_role_arn = aws_iam_role.ecs_task_execution_role.arn
+  container_definitions = jsonencode([
+    {
+      name  = "forecast"
+      image = "${var.ecs_config.docker_image}:${var.ecs_config.docker_version}"
+      #      cpu       = 128
+      #      memory    = 128
+      essential = true
+
+      environment : [
+        {"name": "LOGLEVEL", "value" : "DEBUG"},
+        {"name": "NWP_ZARR_PATH", "value":"s3://${var.s3_nwp_bucket.bucket_id}/data/latest.netcdf"},
+        {"name": "BATCH_SAVE_DIR", "value": "s3://${var.s3_ml_bucket.bucket_id}/"},
```

Comment:
perhaps change name to ML_MODEL_PATH

Reply:
done in [`d80ca41` (#161)](https://github.com/openclimatefix/ocf-infrastructure/pull/161/commits/d80ca41c108b21698b6384ee8712c6dfeae1bef0)